src/components/Page/DefaultSEO/index.tsx

```diff
@@ -76,21 +94,13 @@ const DefaultSEO: React.FC<IDefaultSEOProps> = ({ pageInfo }) => {
       meta={meta}
       link={[
         {
-          rel: 'shortcut icon',
+          rel: 'icon',
           type: 'image/vnd.microsoft.icon',
           href: '/favicon.ico'
```

Comment:
does it have all necessary dimensions? 

also, I wonder if png with certain sizes are needed in some other cases (besides browsers, check the list in the document in the #1119 )

Reply:
I didn't remove these, and I decided to re-add them since their inclusion has minimal impact. They can always be removed again later if the true reason arises.